mu-plugins/blocks/sidebar-container/index.php

```diff
@@ -39,15 +39,19 @@ function init() {
  * @return string Returns the block markup.
  */
 function render( $attributes, $content, $block ) {
-	$back_to_top = sprintf(
-		'<p class="has-small-font-size is-link-to-top"><a href="#wp--skip-link--target">%s</a></p>',
-		esc_html__( '↑ Back to top', 'wporg' )
-	);
+	$back_to_top = $attributes['hasBackToTop']
+		? sprintf(
+			'<p class="has-small-font-size is-link-to-top"><a href="#wp--skip-link--target">%s</a></p>',
+			esc_html__( '↑ Back to top', 'wporg' )
+		)
+		: '';
+	$inlineBreakpoint = $attributes['inlineBreakpoint'];
```

Comment:
```suggestion
	$inline_breakpoint = $attributes['inlineBreakpoint'];
```

PHP variables should be snake case. It looks like we're not running phpcs in the github actions, but you can run it locally with `composer lint mu-plugins/blocks/sidebar-container mu-plugins/blocks/table-of-contents`

Reply:
Fixed in [2aed927](https://github.com/WordPress/wporg-mu-plugins/pull/554/commits/2aed92730dd4ddb3e3e0864f0a09bc5d31da6453)